libraries/cli/include/cli/config_jsons/default.json

```diff
@@ -21,6 +21,8 @@
   "rpc": {
     "http_port": 7777,
     "ws_port": 8777,
+    "gql_http_port": 9777,
+    "gql_ws_port": 6777,
```

Comment:
Would it be better to have:

```
"grapghql": {
    "http_port": 7777,
    "ws_port": 8777,
    "threads_num": 10
  },
```
? (in case graphql http a ws servers are independent from rpc ones ?)